src/ArgentPonyWarcraftClient/WarcraftClient.cs

```diff
@@ -14,20 +13,8 @@ namespace ArgentPonyWarcraftClient
     public class WarcraftClient
     {
         private readonly HttpClient _client;
-
```

Comment:
I felt like these summary docs pointless for private backing fields. But if that's the way you like it then please feel free to change it.

Reply:
It's true.  They provide no value to a library consumer and minimal value to a library developer.